backends/aiger/aiger.cc

```diff
@@ -161,10 +161,10 @@ struct AigerWriter
 			}
 		}
 
-		for (auto bit : input_bits)
+		for (const auto& bit : input_bits)
```

Comment:
Yosys style is (mostly) `const auto &bit` -- can you adopt that instead please? 
I'm also surprised that the compiler can't do this optimisation automatically.

Reply:
done.